src/js/preloadWebComponents.js

```diff
@@ -23,6 +24,11 @@ module.exports = function() {
    if ((androidChrome || iosChrome) && needsWebComponents) {
       // This is requiring webcomponents.js@0.7.24 because that's what was used
       // by the Chromecast framework at the time this was added.
-      require('webcomponents.js'); // eslint-disable-line global-require
+      // We are using webcomponents-lite.js because it doesn't interfere with jQuery as
+      // badly (e.g. it doesn't interfere with jQuery's fix for consistently bubbling
+      // events, see #21). This does not actually polyfill everything Chromecast needs
+      // (i.e. shadow DOM), but we do not use the functionality that is requires shadow
+      // DOM (i.e. `<google-cast-button`)
```

Comment:
What do you think about changing the last sentence to something along the lines of:

> While the "lite" version does not include the shadow DOM polyfills that the Chromecast framework may need for the `<google-cast-button>` component to work properly, this plugin does not use the `<google-cast-button>` component.